File: analyzers/its/sources/ManuallyAddedNoncompliantIssues.VB/IntentionalFindings/S4830.vb
' <Your-Product-Name>
' Copyright (c) <Year-From>-<Year-To> <Your-Company-Name>
'
' Please configure this header in your SonarCloud/SonarQube quality profile.
' You can also set it in SonarLint.xml additional file for SonarLint or standalone NuGet analyzer.

Public Module S4830

    Public Sub TestMethod()
        System.Net.ServicePointManager.ServerCertificateValidationCallback = Function(sender, certificate, chain, sslErrors) True ' Noncompliant (S4830)
    End Sub

End Module

File: analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/IntentionalFindings/S4830.cs
/*
 * <Your-Product-Name>
 * Copyright (c) <Year-From>-<Year-To> <Your-Company-Name>
 *
 * Please configure this header in your SonarCloud/SonarQube quality profile.
 * You can also set it in SonarLint.xml additional file for SonarLint or standalone NuGet analyzer.
 */

namespace IntentionalFindings
{
    public static class S4830
    {
        public static void TestMethod()
        {
            System.Net.ServicePointManager.ServerCertificateValidationCallback += (sender, certificate, chain, sslErrors) => true; // Noncompliant (S4830)
        }
    }
}
